tests/fixtures/debian/control-center/standalone-config.yml

```diff
@@ -79,3 +79,29 @@ services:
       CONTROL_CENTER_STREAMS_CONSUMER_SSL_TRUSTSTORE_PASSWORD: password
     labels:
       - io.confluent.docker.testing=true
+
+  security-config-with-producer-overrides:
+    image: confluentinc/cp-enterprise-control-center:latest
+    command: "bash -c '/etc/confluent/docker/configure && touch /tmp/config-is-done && sleep infinity'"
+    environment:
+      CONTROL_CENTER_ZOOKEEPER_CONNECT: "zookeeper:2181/defaultconfig"
+      CONTROL_CENTER_BOOTSTRAP_SERVERS: "kafka:9092"
+      CONTROL_CENTER_REPLICATION_FACTOR: 1
+      CONTROL_CENTER_COMMAND_TOPIC_REPLICATION: 3
+      CONTROL_CENTER_COMMAND_TOPIC_RETENTION_MS: 1000
+      CONTROL_CENTER_REST_LISTENERS: "http://0.0.0.0:9021,https://0.0.0.0:443"
+      CONTROL_CENTER_REST_SSL_KEYSTORE_LOCATION: "/path/to/keystore"
+      CONTROL_CENTER_CONNECT_TIMEOUT: 30000
+      CONTROL_CENTER_MAIL_ENABLED: "true"
+      CONTROL_CENTER_MAIL_HOST_NAME: "foo.com"
+      CONTROL_CENTER_STREAMS_SECURITY_PROTOCOL: SASL_SS
+      CONTROL_CENTER_STREAMS_SASL_KERBEROS_SERVICE_NAME: kafka
+      CONTROL_CENTER_STREAMS_SSL_KEYSTORE_LOCATION: /path/to/keystore
+      CONTROL_CENTER_STREAMS_SSL_KEYSTORE_PASSWORD: password
+      CONTROL_CENTER_STREAMS_SSL_KEY_PASSWORD: password
+      CONTROL_CENTER_STREAMS_SSL_TRUSTSTORE_LOCATION: /path/to/truststore
+      CONTROL_CENTER_STREAMS_SSL_TRUSTSTORE_PASSWORD: password
+      CONTROL_CENTER_STREAMS_PRODUCER_SECURITY_PROTOCOL: SS
```

Comment:
presumably this is supposed to be `SSL`? and in the test as well? doesn't matter much since it's just an example, but i could imagine someone getting very frustrated while overlooking that one missing letter

Reply:
@ewencp I used the same values that are used above in the `wildcards-config` section. Maybe this is a case of `sed -i` gone wild, but the way I understood it this was probably to ensure we didn't accidentally end up with SSL for some other reason and that those were the actual env values.

Reply:
OK, I don't really understand the reasoning, but given this is all test data I guess it's fine. If we really have a reason for not using SSL, I'd doc it somewhere since I'd probably see this as a typo and just fill in something valid instead if I were updating this.

Reply:
it this is testing an override, can we make it something less confusing like `FAKE_PROTOCOL`?

Reply:
maybe @arrawatia knows why?

Reply:
you mean why `SS` was originally used?

Reply:
@theduderog yes

Reply:
he's out for 2 more weeks.  i'm in favor of changing it for clarity.

Reply:
@theduderog looks like you added some of those in the docs here https://github.com/confluentinc/cp-docker-images/commit/7c93d2536d4d655dc99d44, any reason you used SASL_SS instead of SASL_SSL?

Reply:
lol.  doh!  probably no good reason except by mistake.

Reply:
k, fixed them all and renamed things